### PR TITLE
Update load exception logging

### PR DESCRIPTION
**Pull request by NeonDaniel**
Ensure any plugin load exceptions print the entire exception instead of just a generic message for debugging

### CODE REVIEW

File: ovos_plugin_manager/ocp.py [hunk 1]
```diff
@@ -73,8 +73,8 @@ def extract_stream(self, uri, video=True):
         return meta or {"uri": uri}
 
 
-if __name__ == "__main__":
-    s = StreamHandler()
-    print(s.supported_seis)
-    # ['rss', 'bandcamp', 'youtube', 'ydl', 'youtube.channel.live',
-    # 'pytube', 'invidious', 'm3u', 'pls']
\ No newline at end of file
+# if __name__ == "__main__":
```
Comment (ChanceNCounter) [inline ovos_plugin_manager/ocp.py:76]: What’s going on here?

**Review by ChanceNCounter — COMMENTED** [review]
Reviewed but not tested

File: ovos_plugin_manager/ocp.py (hunk 1 above)
Comment (NeonDaniel) [reply ovos_plugin_manager/ocp.py:76]: Old test code; I commented out when troubleshooting extra OCP init but it wasn't related. I can replace this if wanted

**Review by ChanceNCounter — APPROVED** [review]
Reviewed, still not tested. Looks like boilerplate changes and a linter pass, nothing for CI to miss.

**Pull request merged**